package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "serverless-workflow-vscode-extension",
-	"version": "1.1.0",
+	"version": "1.4.0",
```

Comment:
no need for this change please remove. the version will be auto incremented when the extension is released (via vscode cli)